newIDE/app/src/InstancesEditor/index.js

```diff
@@ -182,7 +186,7 @@ export default class InstancesEditor extends Component<Props> {
     this.pixiRenderer.view.onwheel = (event: any) => {
       const zoomFactor = this.getZoomFactor();
       if (this.keyboardShortcuts.shouldZoom()) {
-        this.zoomOnCursorBy(-event.deltaY / 5000);
+        this.zoomOnCursorBy(getContinuousZoomFactor(-event.deltaY));
```

Comment:
`getContinuousZoomFactor` is a missleading name because it's not continuous, it's discrete but with smaller steps.
What about "getWheelZoomStep" ?